src/prommis/uky/uky_flowsheet.py

```diff
@@ -1973,8 +2001,8 @@ def function(stream):
                 m.fs.solex_cleaner_strip.mscontactor.aqueous_inlet_state[
                     0
                 ].conc_mass_comp.unfix()
-        elif stream == m.fs.precipitator:
-            print(f"Initializing {stream}")
+        elif unit == m.fs.precipitator:
+            print(f"Initializing {unit}")
             try:
                 initializer_bt.initialize(m.fs.precipitator)
             except:
```

Comment:
Why do you need this `try/except` - why not just go to the `except` if you know the first pass will fail?